wadm/lib/wadm/nats.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Wadm.Nats do
+  @moduledoc false
+  require Logger
+
+  # Heavily inspired from the flawless https://github.com/wasmCloud/wasmcloud-otp/blob/main/host_core/lib/host_core/nats.ex
+  def safe_pub(process_name, topic, msg) do
+    if Process.whereis(process_name) != nil do
+      Gnat.pub(process_name, topic, msg)
+    else
+      Logger.warn("Publication on #{topic} aborted - connection #{process_name} is down",
```

Comment:
Should this be `warning`?

Reply:
yeah good catch, I'm used to warn but I noticed the deprecation on the other one :(